docs/source/topics/ref_minimum-system-requirements.adoc

```diff
@@ -1,28 +1,37 @@
 [id="minimum-system-requirements_{context}"]
 = Minimum system requirements
 
+[role="_abstract"]
 {prod} has the following minimum hardware and operating system requirements.
 
 [id="minimum-system-requirements-hardware_{context}"]
 == Hardware requirements
 
-{prod} requires the following system resources:
+{prod} is supported only on AMD64 and Intel 64 processor architectures.
+{prod} does not support the ARM-based M1 architecture.
+{prod} does not support nested virtualization.
+
+{prod} requires the following system resources depending on the desired container runtime:
```

Comment:
```suggestion
{prod} requires the following system resources, depending on the desired container runtime:
```

Reply:
I prefer the latter, it sounds a bit cleaner to me. Thank you!